columnflow/production/cms/seeds.py

```diff
@@ -138,6 +151,17 @@ def deterministic_jet_seeds(self: Producer, events: ak.Array, **kwargs) -> ak.Ar
     return events
 
 
+@deterministic_jet_seeds.setup
+def deterministic_jet_seeds_setup(
+    self: Producer,
+    reqs: dict,
+    inputs: dict,
+    reader_targets: InsertableDict,
+) -> None:
+    # store primes in array
+    self.primes = np.array(primes, dtype=np.uint64)
+
```

Comment:
Since the `init` is (and needs to be) called by every task (and every branch of every workflow) in the entire dependency tree below the producer / calibrator using it, I deemed `setup` to be a better place, which is only called once and in this way, sets up things that are only needed when subsequently calling the actual function payload.

Reply:
I see, that makes sense! Let's keep `setup` then 👍